context/context.go

```diff
@@ -25,8 +26,9 @@ const (
 type Config struct {
 	// Settings.
 
-	Context   netcontext.Context
-	SessionID string
+	Context     netcontext.Context
+	Expectation spec.Expectation
+	SessionID   string
 
 	// TODO we want to track the original input that was provided from the
```

Comment:
The original input needs to be tracked to forward the correct network payload from the output CLG to the input CLG in case a provided expectation was not met.